### DISCUSSION TITLE
STAC Collection and Item without geometry

### DISCUSSION THREAD

**Discussion by matheuspinheirosantos** [radiantearth/stac-spec#1315] [General STAC Q&A] [answered]
The STAC Item specification allow for items with an empty geometry and no bounding box (https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#item-fields). Now the question is, how to represent a STAC Collection that has a single item with no geometry? The main problem is regarding the Spatial Extent Object that is required in the STAC Collection specification (https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#spatial-extent-object). Is there any recommendation in that aspect?

**Comment by m-mohr** [comment, marked as answer]
You can't represent that in STAC. It's the **SpatioTemporal** Asset Catalog as such a spatial and temporal components are usually required. Unlocated Items are a (debatable) exception for some edge-cases where the location is not directly available, see https://github.com/radiantearth/stac-spec/blob/master/best-practices.md#unlocated-items
We excpect for those cases that a rough estimate of the expected location of the Collection can be provided, otherwise STAC might not be the right fit for your usecase. Instead you might want to consider OGC API - Records.
